app/Http/Controllers/Api/v1/User/Controllers/GenerateTokenController.php

```diff
@@ -42,13 +44,33 @@ public function __invoke(Request $request): JsonResponse
         try {
             $user = $this->userLookup->getUpdatedUser($user, $azureUser->principal_name);
         } catch (ClientExceptionInterface) {
+            Log::error([
```

Comment:
Logging isn't pesisted to the database right? So this will not work for us I think.

Also, the array arguments aren't enforced, so this solution is very prone to mistakes

Reply:
"As admin I want to be able to see who has edited the mensa, who signed in who, etc".
Not being able to read it out and link it to a mensa does not fit our requirements

Reply:
How about combining both methods by creating a new Log Channel that writes into the DB instead of a file? I would like to create a system that deviated only slightly from Monolog / facade